### DISCUSSION TITLE
Migrating to an Organization

### DISCUSSION THREAD

**Discussion by acouvreur** [sablierapp/sablier#412] [Announcements]
I want this project to be something that can go a bit further than a "personal project" so I've created the following organization: https://github.com/sablierapp

I want this to be a free and open-source product where it feels more professional.

I'll keep you updated! Feel free to give feedback!

**Comment by gaby** [comment]
@acouvreur In your project GitHub settings there's a section for moving the project to an Org. They will handle all the redirects, etc and you won't lose the project history/issues/commits/etc.

Settings -> Scroll down to Danger Zone -> Transfer

**Comment by acouvreur** [comment]
Done!

Please let me know if any weird behavior should happen!

You should now migrate everything from `acouvreur/sablier` to `sablierapp/sablier` !
